app/lib/actions/ui.js

```diff
@@ -1,6 +1,8 @@
+import { php_escapeshellcmd } from 'php-escape-shell';
```

Comment:
Sorry, thought it was a copy-pasta error! Didn't check the code of php-shell-escape